docs/introduction/tutorials_intro.md

```diff
@@ -258,38 +33,31 @@ These tutorials show you how to install, export a project to and start a debuggi
 
 - [BBC micro:bit, pyOCD and GDB debugging tutorial](../debug-test/debug-microbit.html).
 - [Troubleshooting common issues](../debug-test/troubleshooting-common-issues.html).
-- [Analyzing Mbed OS crash dump tutorial](../debug-test/analyzing-mbed-os-crash-dump.html).
+- [Analyzing Mbed OS crash dump tutorial](../debug-test/analyzing-mbed-os-crash-dump.html) and [crash reporting official example](https://github.com/ARMmbed/mbed-os-example-crash-reporting)
 - [Compile time errors tutorial](../debug-test/compile-time-errors.html).
 - [Debugging using `printf` statements](../debug-test/debugging-using-printf-statements.html).
-- [Error handling API example](../apis/error-handling.html#error-handling-example)
 
 ### Debugging with third party tools
 
 - [Eclipse](../debug-test/third-party-tools.html).
 - [Keil uVision](../debug-test/keil-uvision.html).
 - [Visual Studio](../debug-test/visual-studio-code.html).
 
+## Testing
+
+- [Test your applications or Mbed OS with Greentea](../debug-test/greentea-for-testing-applications.html).
+- [Using the Icetea test framework](../debug-test/icetea.html) and [test applications](../debug-test/icetea-test-applications.html).
 
 ## Bootloader
 
-A [tutorial for creating and using a bootloader](../program-setup/creating-and-using-a-bootloader.html), and [an example bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader).
+- [A tutorial for creating and using a bootloader](../program-setup/creating-and-using-a-bootloader.html).
+- [An official example](https://github.com/ARMmbed/mbed-os-example-bootloader).
+
+## Connecting to the cloud
 
-## Optimizing
+[Pelion Device Management](https://github.com/ARMmbed/mbed-os-example-pelion)
```

Comment:
mbed-os-example-pelion is a lightweigth simple example.
The Pelion example is https://github.com/armmbed/mbed-cloud-client-example

Reply:
Never sure which version of the OS they use in their example; I'll keep using just the Mbed OS official

Reply:
BTW, do you know whether the AWS example can be linked?

Reply:
Not yet